packages/definitions/src/fieldType.ts

```diff
@@ -79,6 +80,21 @@ export const FIELD_TYPE = Object.freeze({
   GEOMETRY_MULTILINESTRING: 'GEOMETRY_MULTILINESTRING',
 })
 
+/**
+ * Component to use for a field
+ */
+export const FIELD_COMPONENT = Object.freeze({
+  INPUT_TEXT: 'INPUT_TEXT',
+  INPUT_PASSWORD: 'INPUT_PASSWORD',
+  INPUT_NUMBER: 'INPUT_NUMBER',
+  INPUT_FLOAT: 'INPUT_FLOAT',
+  INPUT_CURRENCY: 'INPUT_CURRENCY',
+  INPUT_DATE: 'INPUT_DATE',
+  INPUT_DATETIME: 'INPUT_DATETIME',
+  INPUT_EMAIL: 'INPUT_EMAIL',
+  INPUT_UUID: 'INPUT_UUID',
+})
+
 export type DB_TYPE = pgDbTypes | sqliteDbTypes
 export type DB_DIALECT = 'pg' | 'sqlite3'
```

Comment:
Another suggestion: use the diminutive `pgsql` for PostgreSQL, instead of `pg`. Not a major topic but I think `pgsql` evoke more spontaneously "PostgreSQL" for me than simply `pg`.

Reply:
This is related to `knex`, check the documentation here https://knexjs.org/guide/#configuration-options
They use `pg` for postgres connections.
And we respect the glossary specified by `knex`.
But I didn't find any type in the `knex` library.
So I create this one, to be sure of the string itself.

Reply:
> They use pg for postgres connections. And we respect the glossary specified by knex.

Ok! I understand :slightly_smiling_face: